apps/vr-tests/screener.local.config.js

```diff
@@ -1,6 +1,8 @@
 module.exports = {
   projectRepo: 'microsoft/fluentui',
   storybookConfigDir: '.storybook',
+  storybookVersion: '5',
```

Comment:
Could you add a comment with a link to this PR so it's easy to find the background of why this is here?